code/modules/antagonists/eldritch_cult/eldritch_antag.dm

```diff
@@ -264,3 +292,115 @@
 	if(!cultie)
 		return FALSE
 	return cultie.total_sacrifices >= target_amount
+
+/datum/objective/ascend
+	name = "ascend"
+
+/datum/objective/ascend/update_explanation_text()
+	..()
+	explanation_text = "Appease the Gods and ascend."
```

Comment:
update explanation text is for things that have dynamic explanation texts EG (Sacrafice 5-7 people). Anything that doesn't change can just be set in the datum itself.